index.html

```diff
@@ -459,8 +459,30 @@ <h5 data-i18n="footer.in-touch.header">Keep in Touch</h5>
         });
 
         $.when(ajaxRequest, i18nLoaded).done(function (data) {
-            var build = data[0][0],
-                buildName = build.versionString;
+            var allBuilds = data[0],
+                build = data[0][0]; // default initialization with latest build
+
+            // Find the applicable build for current platform
+            // If current platform is unsupported (OTHERS), then take the last common release
+            // If a build entry doesn't have 'platforms' key -> Older format for all supported OS
+            for (var buildIndex = 0; buildIndex < allBuilds.length; buildIndex++) {
+                build = allBuilds[buildIndex];
+                if (!build.platforms) { // Older format detected - Applicable to all platforms
+                    break;
+                } else {
+                    if (OS === "OTHER") { // Unsupported platform detected
+                        if (Object.keys(build.platforms).length == 4) {
```

Comment:
shouldn't this be === . Also what does this check mean?

Reply:
We are avoiding a type check as both the LHS and RHS are know to be numbers.

Reply:
shouldn't (build.platforms[OS]) be the first check ? Also, how does checking for length of keys the build.platforms help ? what if support for another platform is added in future ?